provisioning_client/adapters/hsm_client_http_edge.c

```diff
@@ -140,13 +140,25 @@ static int read_and_parse_edge_uri(HSM_CLIENT_HTTP_EDGE* hsm_client_http_edge)
         }
         else if (hsm_client_http_edge->workload_protocol_type == WORKLOAD_PROTOCOL_TYPE_UNIX_DOMAIN_SOCKET)
         {
+            const char* socket_path = workload_uri + unix_domain_sockets_prefix_len;
+
+#ifdef WIN32
```

Comment:
We're not doing it this way anywhere else in the C SDK, so I'm open to feedback about how best to make sure we only strip the forward slash on Windows devices.